lib/Providers/Time/NTPTimeProvider.php

```diff
@@ -47,7 +47,7 @@ public function getTime()
 
             // Interpret response
             $data = unpack('N12', $recv);
-            $timestamp = (int) sprintf('%u', $data[9]);
+            $timestamp = (int)sprintf('%u', $data[9]);
```

Comment:
I find a space after the type coercion much more readable

Reply:
Another PHP-CS-Fixer rule.